docs/user/api-server.md

```diff
@@ -0,0 +1,27 @@
+# Turbinia API Server
```

Comment:
It might be good to add a comment here somewhere that mentions that the API server requires direct access to the Turbinia output directory.

Reply:
Done